lib/src/source/git.dart

```diff
@@ -72,6 +75,32 @@ class GitSource extends CachedSource {
             'string.');
       }
       path = descriptionPath;
+
+      // TODO: can we avoid relying on key presence?
+      if (description.containsKey('tag_pattern')) {
```

Comment:
What about simply using `tags` and then support one or more patterns?
```yaml
dependencies:
  foo:
    git:
      url: https://foo.bar.git/repo
      tags: v*
```
and
```yaml
dependencies:
  foo:
    git:
      url: https://foo.bar.git/repo
      tags:
       - v*
       - foo-v*
```

or maybe not.

Reply:
Yeah - that might work